src/java/voldemort/store/metadata/MetadataStore.java

```diff
@@ -1168,6 +1168,16 @@ public void initNodeId(int nodeId) {
         }
     }
 
+    public static StoreDefinition getStoreDef(String store, MetadataStore metadataStore) {
+        StoreDefinition def = null;
+        if(SystemStoreConstants.isSystemStore(store)) {
```

Comment:
This function could return null if store is treated as system store, but the definition doesn't exist, since 'SystemStoreConstants.isSystemStore' only checks the store name prefix.
Normally, it won't happen, but I think it is better to throw VeniceException when it is null to keep the logic consistent.